conf/test_screenpathogen.config

```
@@ -25,13 +25,13 @@ params {
 
     // Input data
     input                       = params.pipelines_testdata_base_path + 'samplesheet.csv'
-    pathogens_genomes           = params.pipelines_testdata_base_path + 'reference/reference.fasta'
+    pathogens_genomes           = params.pipelines_testdata_base_path + 'reference/reference.fasta.gz'
     accession2taxid             = params.pipelines_testdata_base_path + 'reference/accession2taxid.map'
     blastn_db                   = params.pipelines_testdata_base_path + 'blastdb/blastn/blastn_testdb.tar.gz'
     blastx_db                   = params.pipelines_testdata_base_path + 'blastdb/blastx/diamond_testdb.dmnd'
 
     // Extract reads
-    perform_extract_reads       = false
+    perform_verify_species       = false
```

```suggestion
    perform_verify_species     = false
```

The alignment does not appear correct.